ultrack/cli/export.py

```diff
@@ -187,6 +193,45 @@ def trackmate_cli(
     to_trackmate(config, output_path, overwrite)
 
 
+@click.command("geff")
+@click.argument(
+    "database_path",
+    type=click.Path(path_type=Path, exists=True),
+)
+@click.option(
+    "--output-path",
+    "-o",
+    required=False,
+    type=click.Path(path_type=Path),
+    default=None,
+    help=(
+        "Geff (Graph Exchange File Format) output path. "
+        "If not provided, saves to same directory as database with '_geff.geff' extension."
+    ),
+)
+@overwrite_option()
+def geff_cli(
+    database_path: Path,
+    output_path: Optional[Path],
+    overwrite: bool,
+) -> None:
+    """
+    Exports tracking results to Geff (Graph Exchange File Format) format.
+    """
+    if output_path is None:
+        # Generate output path from database path
+        output_path = database_path.parent / f"{database_path.stem}_geff.geff"
```

Comment:
```suggestion
        output_path = database_path.parent / f"{database_path.stem}.geff"
```
What do you think of removing the `_geff`?
Otherwise it's `<NAME>_geff.geff`.